MdeModulePkg/Library/DxeCapsuleLibFmp/DxeCapsuleLib.inf

```diff
@@ -51,6 +52,7 @@
   DisplayUpdateProgressLib
   FileHandleLib
   UefiBootManagerLib
+  BlParseLib

```

Comment:
Would it be possible to do the parsing in `BlSupportPei` and create HOB(s) with `EFI_HOB_TYPE_UEFI_CAPSULE` type there? Adding dependencies to anything in `MdeModulePkg` would increase maintenance burden.

Reply:
Yes and that's a better place for it.  New implementation: addfc78fd93743318ab089dc613e3e10caf4bba4.